tangelo/toolboxes/post_processing/histogram.py

```diff
@@ -18,7 +18,7 @@
 
 from collections import Counter
 
-from tangelo.linq import Simulator
+from tangelo.linq.simulator_base import SimulatorBase
```

Comment:
Let's directly import the function and not that class. (there are probably a few other files in your PR where this happens)